app/_locales/en_GB/messages.json

```diff
@@ -197,7 +197,7 @@
     "message": "Add friends and addresses you trust"
   },
   "addHardwareWallet": {
-    "message": "Add hardware wallet"
+    "message": "Hardware wallet"
```

Comment:
For future reviewers: This is expected and aligned with the corresponding designs. Basically, we now drop the "Add " prefix (NOTE: This is true for other `addNew*`)